ubuntu.json

```diff
@@ -45,6 +45,26 @@
     "key_url": "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x16126D3A3E5C1192"
   },
   {
+    "alias": "fpcomplete-stable-precise",
+    "sourceline": "deb http://download.fpcomplete.com/ubuntu/precise stable main",
+    "key_url": "http://download.fpcomplete.com/ubuntu/fpco.key"
+  },
+  {
+    "alias": "fpcomplete-stable-trusty",
+    "sourceline": "deb http://download.fpcomplete.com/ubuntu/trusty stable main",
+    "key_url": "http://download.fpcomplete.com/ubuntu/fpco.key"
+  },
+  {
+    "alias": "fpcomplete-stable-utopic",
```

Comment:
Please remove this and the vivid one, unless they are known to work with Precise and Trusty.